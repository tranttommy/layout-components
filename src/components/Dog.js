import React from 'react';

export default function Dog() {
  const dog = {
    name: 'Fafa',
    age: 5,
    weight: '45 lbs'
  };

  return (
    <dl>
      <dt>Name</dt>
      <dd>{ dog.name }</dd>

      <dt>Age</dt>
      <dd>{ dog.age }</dd>

      <dt>Weight</dt>
      <dd>{ dog.weight }</dd>
    </dl>
  );
}
