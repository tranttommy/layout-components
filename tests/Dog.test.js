import React from 'react';
import { shallow } from 'enzyme';
import Dog from '../src/components/Dog.js';

describe('Dog component', () => {
  it('renders a Dog', () => {
    const wrapper = shallow(<Dog />);
    expect(wrapper).toMatchSnapshot();
  });
});
