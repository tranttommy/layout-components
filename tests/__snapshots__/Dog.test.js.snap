// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dog component renders a Dog 1`] = `
<dl>
  <dt>
    Name
  </dt>
  <dd>
    Fafa
  </dd>
  <dt>
    Age
  </dt>
  <dd>
    5
  </dd>
  <dt>
    Weight
  </dt>
  <dd>
    45 lbs
  </dd>
</dl>
`;
